packages/smooth_app/lib/pages/product/common/product_list_modal.dart

```diff
@@ -218,6 +218,17 @@ class _ProductListPageState extends State<ProductListPage>
         actionModeSubTitle:
             Text(appLocalizations.compare_products_appbar_subtitle),
         actionModeActions: <Widget>[
+          IconButton(
+            onPressed: () => _toggleAllSelection(),
+            tooltip: _allItemsSelected
+                ? appLocalizations.compare_products_unselect_all_items
+                : appLocalizations.compare_products_select_all_items,
+            icon: Icon(
+              _allItemsSelected
+                  ? Icons.library_add_check_rounded
```

Comment:
That icon ("check") is not really appropriate, is it?

Reply:
It will be a plus, when we want to add all.
And the check is just like on GMail for instance:

<img width="315" alt="Screenshot 2023-08-04 at 13 38 18" src="https://github.com/openfoodfacts/smooth-app/assets/246838/1847cc73-c1fa-41e2-aceb-673135b70e75">